qubesadmin/utils.py

```diff
@@ -115,3 +115,29 @@ def updates_vms_status(qvm_collection):
             # "mixed"
             return None
     return status
+
+# Helper function that returns a list of all the places a given VM is used in.
+# Output is a list of tuples (property_holder, property_name), with None as
+# property_holder for global properties
+
+
+def vm_usage(app, reference_vm):
```

Comment:
Could you make this function name more specific? IMO `vm_usage` could be confused with disk usage.